### PR TITLE
Change: Manifest Json

### PR DESCRIPTION
**Pull request by martinmckenna**
## Description

🚨 disclaimer: I have no idea if this works on Android

Updates `manifest.json` to get Cloud closer to a PWA. We're never really going to get there until we start looking at service workers, but this is a good start in terms of pinning the app to your phone's dashboard.

## Recommended Reading

[MDNs guide to manifest.jsons](https://developer.mozilla.org/en-US/docs/Web/Manifest)

[Why is the apple PNG icon not transparent?](https://makandracards.com/makandra/26757-do-not-use-transparent-pngs-for-ios-favicons)

[Why do we have both 512px and 192px PNGs?](https://developers.google.com/web/fundamentals/web-app-manifest)

## To test

1. Run Chrome Dev Tools lighthouse for PWAs. Compare this branch vs. develop

2. On either iOS or Android, [pin the webpage to your home screen](https://www.howtogeek.com/196087/how-to-add-websites-to-the-home-screen-on-any-smartphone-or-tablet/) and see how the icons look

## Preview

unnamed

### CODE REVIEW

File: packages/manager/public/index.html [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-  <script src="https://static.ada.support/embed.be6a6728.min.js" charset="utf-8"></script>
-</html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="theme-color" content="#000000">
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/assets/logo-192-apple.png">
```
Comment (martinmckenna) [inline packages/manager/public/index.html]: the diff isn't great on this one. The only lines I changed was this one and added https to the crazyegg script.

File: packages/manager/public/manifest.json [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+  "short_name": "Linode",
+  "name": "Linode Cloud Manager",
+  "categories": [
+    "cloud hosting",
+    "cloud",
+    "linode",
+    "compute"
+  ],
+  "mainifest_version": 2,
+  "version": "0.72.0",
```
Comment (Jskobos) [inline packages/manager/public/manifest.json]: Does this get bumped automatically with lerna publish?

Comment (martinmckenna) [reply packages/manager/public/manifest.json]: it does not, but I don't think it's that big of a deal.

Comment (martinmckenna) [reply packages/manager/public/manifest.json]: actually, web app don't support the version key at all anyway. Removing...

**Pull request merged**